sympy/core/function.py

```diff
@@ -1427,7 +1427,7 @@ def __new__(cls, expr, *variables, **kwargs):
         if isinstance(expr, Derivative):
             variable_count = list(expr.variable_count) + variable_count
             expr = expr.expr
-            return _derivative_dispatch(expr, *variable_count, **kwargs)
+            return self.func(expr, *variable_count, **kwargs)
```

Comment:
This is a classmethod so there is no `self`.

Reply:
I am a beginner in Open source . I want to contribute to SymPy . Can you please help me with this ?